Project-Aurora/Project-Aurora/Settings/VariableRegistry.cs

```diff
@@ -1,5 +1,7 @@
 ﻿using Aurora.Utils;
+
 using Newtonsoft.Json;
+
 using System;
```

Comment:
What's with these sort of unnecessary changes? We've not got any whitespace in the imports in other files.

Reply:
https://github.com/antonpup/Aurora/pull/1757#discussion_r336581440